experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -87,10 +87,10 @@ export abstract class InstrumentationBase<
       return wrap(moduleExports, name, wrapper);
     } else {
       const wrapped = wrap(Object.assign({}, moduleExports), name, wrapper);
-
-      return Object.defineProperty(moduleExports, name, {
+      Object.defineProperty(moduleExports, name, {
         value: wrapped,
       });
+      return wrapped;
```

Comment:
Reviewer note: From a quick check, I didn't see any core or contrib-repo instrumentations using the return value of `this._wrap`.  Arguably this is a "fix" for `@opentelemetry/instrumentation`, but I didn't mention it in the CHANGELOG.